libraries/math/src/approximations.rs

```diff
@@ -41,6 +41,31 @@ pub fn sqrt<T: PrimInt + CheckedShl + CheckedShr>(radicand: T) -> Option<T> {
     Some(result)
 }
 
+/// Calculate the normal cdf of the given number
+///
+/// Code lovingly adapted from the excellent work at:
+///
+/// <https://www.hrpub.org/download/20140305/MS7-13401470.pdf>
+///
+/// The algorithm is based on the implementation in the paper above.
+#[inline(never)]
+pub fn f32_normal_cdf(argument: f32) -> Option<f32> {
```

Comment:
nit: This function returns an `Option` but doesn't do any checked math.  Seems like it can just return `f32` in that case, right?